src/django/api/tests/test_sectors_view.py

```diff
@@ -0,0 +1,143 @@
+from api.constants import FacilityClaimStatuses
+from api.models.contributor.contributor import Contributor
+from api.models.facility.facility import Facility
+from api.models.facility.facility_claim import FacilityClaim
+from api.models.facility.facility_list import FacilityList
+from api.models.facility.facility_list_item import FacilityListItem
+from api.models.sector import Sector
+from api.models.sector_group import SectorGroup
+from api.models.source import Source
+from api.models.user import User
+from api.tests.facility_api_test_case_base import FacilityAPITestCaseBase
+from rest_framework import status
+
+from django.contrib.gis.geos import Point
+from django.urls import reverse
```

Comment:
Please stick to [PEP 8](https://peps.python.org/pep-0008/#imports) (the official style guide for Python code) when organizing imports.
From the documentation ->>>

> Imports should be grouped in the following order:
> 
> - Standard library imports.
> - Related third party imports.
> - Local application/library specific imports.
> 
> You should put a blank line between each group of imports.